### ISSUE TITLE
Missing LICENSE file

### ISSUE THREAD

**Issue by mado89** [alessandroBombelli/pyHoops#49]
Although the readme states the project served under MIT license this file is missing (required by the journal conventions: "Does the repository contain a plain-text LICENSE file with the contents of an OSI approved software license?")

https://github.com/openjournals/joss-reviews/issues/1845

**Comment by alessandroBombelli** [comment]
> Although the readme states the project served under MIT license this file is missing (required by the journal conventions: "Does the repository contain a plain-text LICENSE file with the contents of an OSI approved software license?")
> 
> [openjournals/joss-reviews#1845](https://github.com/openjournals/joss-reviews/issues/1845)

Hi @mado89, maybe I am missing something, but I added a MIT license I think ---> This project is licensed under the MIT License.

**Comment by mado89** [comment]
Yes that information is present. But afaik I understand it you need a copy of the license present in your repo and not only the reference.
Maybe @danielskatz can clarify.

**Comment by danielskatz** [comment]
Yes, a LICENSE file is needed

**Comment by alessandroBombelli** [comment]
> Yes, a LICENSE file is needed

Added license file
